sonic_platform_base/sonic_xcvr/api/xcvr_api.py

```diff
@@ -75,8 +75,8 @@ def get_transceiver_info_firmware_versions(self):
         Retrieves active and inactive firmware versions of the xcvr
 
         Returns:
-            A list with active and inactive firmware versions of the xcvr
-            [active_firmware, inactive_firmware]
+            A dictionary containing the active and inactive firmware versions of the transceiver
```

Comment:
@AnoopKamath please note this change for your PR...

Reply:
Ack. Thanks @prgeor